assets/js/googlesitekit/datastore/user/nonces.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * `core/user` data store: nonces.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import Data from 'googlesitekit-data';
+import { CORE_USER } from './constants';
+import { createFetchStore } from '../../data/create-fetch-store';
+const { createRegistrySelector } = Data;
+
+// Actions
+const RECEIVE_NONCES = 'RECEIVE_NONCES';
+
+const fetchGetNoncesStore = createFetchStore( {
+	baseName: 'getNonces',
+	controlCallback: () => {
+		return API.get( 'core', 'user', 'nonces', undefined, {
+			useCache: false,
+		} );
+	},
+	reducerCallback: ( state, nonces ) => ( {
+		...state,
+		nonces,
+	} ),
+} );
+
+const baseInitialState = {
+	nonces: null,
```

Comment:
Is there any reason to initialise this as `null` instead of an empty object? Initialising as an empty object might give developers a better sense of what to expect here, and since `nonces[ nonce ]` will still return `undefined` when loading, the experience would be the same 🤔

Reply:
When using `null` it's easier check if it's falsy. Using undefined could also work.
So we can use `if( nonces )` instead of `if ( nonces && Object.keys( nonces ).length )`

I verified with other fetchStores. Some of them use `null`, some `undefined` and some empty objects. So I'm not sure what's the recommended way here.